packages/aws-cdk-lib/aws-events-targets/lib/appsync.ts

```diff
@@ -0,0 +1,81 @@
+import { addToDeadLetterQueueResourcePolicy, bindBaseTargetConfig, singletonEventRole, TargetBaseProps } from './util';
+import * as appsync from '../../aws-appsync';
+import * as events from '../../aws-events';
+import * as iam from '../../aws-iam';
+
+/**
+ * Customize the AppSync GraphQL API target
+ */
+export interface AppSyncGraphQLAPIProps extends TargetBaseProps {
+
+  /**
+   * The GraphQL operation; that is, the query, mutation, or subscription
+   * to be parsed and executed by the GraphQL service.
+   */
+  readonly graphQLOperation: string;
+
+  /**
+   * The variables that are include in the GraphQL operation.
+   */
+  readonly variables: events.RuleTargetInput;
```

Comment:
```suggestion
  readonly variables?: events.RuleTargetInput;
```
Can we expect GraphQL queries without variables?

Reply:
changed to optional